### DISCUSSION TITLE
[자기소개] - 8/31 이전 가입 멤버들

### DISCUSSION THREAD

**Discussion by HangryDev** [Pseudo-Lab/Anxious-Lookout#2] [Announcements]
[이 글은 삭제 예정입니다!] 

저는 정한결이고, 이런 사람입니다. 
https://pseudo-lab.com/2aa44fc7935b418780496b5bef17af41?pvs=4

**Comment by hojie11** [comment]
(펑)

**Reply by jungyu0413** [reply]
안녕하세요. 경영학부 졸업 이후, 인공지능 분야에서 인턴 등으로 근무하다가 현재는 대학원 과정에서 연구하고 있습니다.
감사합니다!

**Comment by Hitdahit2** [comment]
(접수 완료! 펑)

**Comment by Garurubi** [comment]
(접수 완료! 펑)

**Comment by hidden719** [comment]
(접수 완료! 펑)

**Comment by kuongee** [comment]
(접수 완료! 펑)

**Comment by kys42** [comment]
(접수 완료! 펑)

**Comment by YunSeok-Kang** [comment]
(접수 완료! 펑)

**Comment by short-armed-ape** [comment]
(접수 완료! 펑)

**Comment by cjkangme** [comment]
(접수 완료! 펑)

**Comment by yeoamlog** [comment]
(접수 완료! 펑)

**Comment by hyeon9698** [comment]
(접수 완료! 펑)

**Comment by JENNYCAKE** [comment]
(접수 완료! 펑)

**Comment by JeongSuJin** [comment]
(접수 완료! 펑)

**Reply by JeongSuJin** [reply]
[자기소개] 
안녕하세요! 컴공 졸업 후 직장인으로 5년 반 넘게 일하면서 야간대학원에서 곧 데이터 사이언스 석사 취득을 하는 JeongSuJin 입니다! 실력은 매우 작고 귀여운 느낌입니다.. 말하는 감자 수준입니다.
AI 관련 사이드 프로젝트에 관심이 있습니다. 여러분 함께해요!!

**Comment by johnjaejunlee95** [comment]
(접수 완료! 펑)

**Reply by johnjaejunlee95** [reply]
안녕하세요!! 
저는 현재 UNIST 인공지능대학원에서 박사과정 3년차 이재준입니다. 
관심분야로는 Meta-Learning, Multi-Modal Learning 등에 대해서 연구하고 있습니당 :) 
가짜연구소 들어온지 1년 반 정도 됐고, 가끔 스터디 일원으로써, 블로깅 채널 등에서 출몰하고 있습니다 ㅎㅎ
집단 지성을 믿는 사람이라 계속 함께 정보 등을 주고 받으면 좋을 것 같습니다!! 
감사합니다!!

블로그: https://johnjaejunlee95.github.io/
링크드인: https://www.linkedin.com/in/johnjaejunlee95/
깃헙: https://github.com/johnjaejunlee95

**Comment by Jun0zo** [comment]
(접수 완료! 펑)

**Comment by yunha-p** [comment]
(접수 완료! 펑)

**Reply by yunha-p** [reply]
[자기소개] 경영학, 산업공학 학사이고, 올 8월에 인간공학 석사 졸업예정입니다! 사용성/UX에 관심이 많고, 요새는 파이썬으로 머신러닝을 공부해보고 있습니다😇

**Comment by HangryDev** [comment]
저는 이런 사람입니다:

https://pseudo-lab.com/2aa44fc7935b418780496b5bef17af41?pvs=4

**Comment by AlanulousL** [comment]
안녕하세요!!
저는 현재 식자재 구매 플랫폼에서 데이터 분석가로 일하고 있는 이원진이라고 합니다.
분석가로 커리어를 쌓은지는 4년차이지만 제대로 하지 못하고 시간만 지난 느낌이 있어 지식쪽으로도 실무쪽으로 많은 갈증을 느끼고있습니다.
데이터 캠프 라이센스만 받아서 잘 사용하고 있었는데 본격적인 활동을 시작해보고 싶습니다.
감사합니다!!

링크드인: https://www.linkedin.com/in/alan-wonjin-lee/

**Reply by gdone9009** [reply]
닉네임 강동원 (실명 ㅈㅊㅅ) 마법학교 입교합니다. 명칭이 너무 좋네요. 경영전공이고, 최근 인공지능 빅데이터 공부하고 있는데, 성과가 좋지 않습니다. 많이 배우고 제가 나눌수 있는 지식은 공유하고 싶구요. 컴퓨터그래픽 중급, 3d프린터운용 및 3d디자인 초급, uxui디자인 수업듣고 입문수준입니다. 잘 부탁드립니다.

**Comment by twirlined** [comment]
안녕하세요! 컴공 학사를 졸업하고 석사과정 휴학 후 스타트업에서 연구 기획 업무를 하고 있는 고예송입니다. 연구실 환경이 아닌 현업에서의 AI 활용에 대해 많이 배우고 싶습니다. 감사합니다.

**Comment by spyhop23** [comment]
안녕하세요! 데이터사이언스 석사 과정중인 김지연입니다. 학교 수업 외에도 더 많이 배우고 경험하고 싶습니다. 감사합니다.

**Comment by darbyyyy** [comment]
안녕하세요. 현업에서 AI 개발하는 박재용입니다.
여러 정보 + 그를 대하는 시각들을 함께 나누고 싶어요.
과거 computer vision, 최근 LLM을 주로 다루고 있습니다.

**Reply by HangryDev** [reply]
https://open.kakao.com/o/s9284d9g

안녕하세요 재용님 여기로 인사주세요!

**Comment by doosanpark** [comment]
안녕하세요!
현재 개발자이고 AI 엔지니어로 커리어 전환하기 위한 시도 중에 있습니다.
서강대학교 AI.SW대학원에서 데이터사이언스 인공지능학과 재학중에 있습니다.
RAG 서비스를 말든 경험이 있습니다!
